### PR TITLE
Use generator plugins

### PR DESCRIPTION
**Pull request by izumin5210**
## WHY
merge into #99

### CODE REVIEW

File: pkg/grapicmd/cmd/generate.go [hunk 1]
```diff
@@ -2,124 +2,62 @@ package cmd
...
+		RunE: func(_ *cobra.Command, args []string) error {
+			return repo.Run(context.TODO(), t.Name(), append([]string{subCmd}, args...)...)
 		},
 	}
-
-	cmd.PersistentFlags().BoolVarP(&skipTest, "skip-test", "T", false, "Skip test files")
-	cmd.PersistentFlags().StringVar(&resourceName, "resource-name", "", "ResourceName to be used")
-
+	cmd.SetHelpFunc(func(_ *cobra.Command, _ []string) {
+		repo.Run(context.TODO(), t.Name(), subCmd, "--help")
```
Comment (watchbunny) [inline pkg/grapicmd/cmd/generate.go]: **[errcheck]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
	repo.Run(context.TODO(), t.Name(), subCmd, "--help")

File: pkg/grapicmd/context.go [hunk 2]
```diff
@@ -48,6 +49,11 @@ type BuildConfig struct {
 
 // Init initializes the runtime context.
 func (c *Ctx) Init() error {
+	if c.RootDir == "" {
+		dir, _ := os.Getwd()
```
Comment (watchbunny) [inline pkg/grapicmd/context.go]: **[errcheck]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
	dir, _ := os.Getwd()

File: pkg/gencmd/generator.go [hunk 3]
```diff
@@ -59,10 +59,13 @@ func (g *generatorImpl) Generate(params interface{}) error {
 		}
 
 		path := g.rootDir.Join(e.Path)
+		dir := filepath.Dir(path)
 
-		err := g.fs.MkdirAll(path, 0755)
-		if err != nil {
-			return errors.Wrapf(err, "failed to create directory")
+		if ok, _ := afero.DirExists(g.fs, dir); !ok {
```
Comment (watchbunny) [inline pkg/gencmd/generator.go]: **[errcheck]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
	if ok, _ := afero.DirExists(g.fs, dir); !ok {

File: pkg/grapicmd/context.go (hunk 2 above)
Comment (izumin5210) [reply pkg/grapicmd/context.go]: ok

**Pull request merged**